website/docs/docs/dbt-cloud-apis/metadata-api.md

```diff
@@ -1,44 +1,85 @@
+- The [dbt Metadata API](#use-cases) &mdash;  Use it to fetch metadata related to the state and health of your dbt project. 
+- The [dbt Cloud Administrative API](/docs/dbt-cloud-apis/admin-cloud-api) &mdash; Use it to administrate a dbt Cloud account. For example, manually retrieve the JSON artifact files, check job status, and run jobs. 
 
-## Prerequisites
+<Snippet src="metadata-api-prerequisites" />
 
-- You must have a [multi-tenant](/docs/deploy/regions-ip-addresses) account and [Team or Enterprise plans](https://www.getdbt.com/pricing/)
-- Your projects must be on dbt version v0.19.0 or higher. Refer to [Version migration guides](/guides/migration/versions) to upgrade
     
 ## Use cases
 
-Use the Metadata API to solve the following use cases:
+Use the Metadata API directly or through an integrated tool. It powers some experiences in dbt Cloud and can help solve [use cases](/docs/dbt-cloud-apis/metadata-use-case-guides) that benefit from [webhooks](/docs/deploy/webhooks) for accessing the latest project and run information:
+
+- [**Discovery**](/docs/dbt-cloud-apis/metadata-use-case-guides#discovery)* &mdash; Find and understand dbt assets to analyze in integrated tools using information like model and metric definitions, column info, and lineage. One example of this is the [dbt Semantic Layer integration](/guides/dbt-ecosystem/sl-partner-integration-guide). 
+- [**Quality**](/docs/dbt-cloud-apis/metadata-use-case-guides#quality)* &mdash; Make sure users have correct and up-to-date data for their analyses by monitoring test failures, source freshness, run status, exposures, and dependencies.
+- [**Operations**](/docs/dbt-cloud-apis/metadata-use-case-guides#operations)* &mdash;  Help data teams run dbt efficiently and reduce costs by using historical run data, including information like model build time and run counts.
+
+*_Refer to [Using the Metadata API](/docs/dbt-cloud-apis/metadata-use-case-guides) for more info; additional detailed use case and integration guides are coming soon_.
+
+
+## Product roadmap
+
+The 2023 Metadata API roadmap outlines three main uses: discovery, quality, and operation, with specific initiatives planned for each quarter:
+
+<!--- tabs for discovery, quality, operations --->
+<Tabs>
+
+<TabItem value="discovery" label="Discovery">
+
+To improve discovery experiences, we’ll make it easier for API users to access the latest production state of a project. This is our primary focus in the first half of 2023. 
+
+- Q2 &mdash; dbt v1.6: Use public models to enable [multi-project deployments](https://github.com/dbt-labs/dbt-core/discussions/6725) and access the global lineage
+- Q2 &mdash; dbt v1.6: New and revised model and entity endpoints to unlock Semantic Layer use cases
+- Q2 &mdash; Pagination for manageable responses and performance when querying long lists of dbt objects
+- Q2 &mdash; Instead of querying per job, get the DAG state based on the latest production runs of its environment.
```

Comment:
@mirnawong1 Let's please expand a bit (text below) and move this one to the top. We're on track to release end of Q1. I'll share doc and use case updates in early April. 

"- Q1 &mdash; Instead of querying per job/run, get the project's logical state (definitions), execution results, and applied state (the output of execution - what exists in the database) based on the latest production runs of its environment."